ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -2166,19 +2166,20 @@ public class BrowserViewController: UIViewController {
       DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(500)) {
         self.screenshotHelper.takeScreenshot(tab)
       }
-    } else if let webView = tab.webContentView {
+    } else if tab.isWebViewCreated {
       // Ref #2016: Keyboard auto hides while typing
       // For some reason the web view will steal first responder as soon
       // as its added to the view heirarchy below. This line fixes that...
       // somehow...
-      webView.resignFirstResponder()
+      // FIXME: Do we still need this?
```

Comment:
https://github.com/brave/brave-ios/pull/2057

Original PR.

Reply:
Yeah left a FIXME to re-test, otherwise will have to add another method to WebViewProxy to enable resigning as well